components/Home.vue

```diff
@@ -18,8 +18,55 @@
 .home-container {
   height: calc(100vh - 88px);
 
+  @media (max-width: 1023px) {
+    height: auto;
+  }
+
+  @media (width: 1024px) and (height: 1366px) {
+    background-image: url('../assets/img/imagem_home_rocket.jpg');
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: cover;
+
+    .n-link-button {
+      background-color: #fff;
+      background-image: none;
+      color: #110721;
+    }
+  }
+
+  &__image {
+    background-image: url('../assets/img/imagem_home_rocket.jpg');
+    background-position: 0 -100px;
+    background-repeat: no-repeat;
+    background-size: cover;
+    height: 500px;
+    width: 100%;
+
+    @media (max-width: 1023px) {
+      height: 300px;
+    }
+
+    @media (width: 1024px) and (height: 1366px) {
```

Comment:
acho melhr n definir height, se alguem entrar com essa msm rsolução mas n for um tablet como fica?

Reply:
Igual a um tablet